pkg/providers/korifi/test_data/test_config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: EXAMPLECRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURCVENDQWUyZ0F3SUJBZ0lJTld2ZEhlbUZYekl3RFFZSktvWklodmNOQVFFTEJRQXdGVEVUTUJFR0ExVUUKQXhNS2EzVmlaWEp1WlhSbGN6QWVGdzB5TlRBME1UWXdPREV3TVRGYUZ3MHpOVEEwTVRRd09ERTFNVEZhTUJVeApFekFSQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBQTRJQkR3QXdnZ0VLCkFvSUJBUURCbTRzVk5RMWRZNEpQdDdWQzJSQjZDdmJtUzRVei9MUzdhNHdhZkNLVzBFWWdMa0MrN3l2UE1TTEkKeXR2VjVVYUlEVXBEeE4zQkNyemlwQUFRbXVjWnVKNXZoTzRBN2QreVpReVhjSjN1R3NMbUdEc2o3ek5ZMHpWZQpXL1cwT1R2YmNpMTFqaUYwOEdQNzBLZmVpaFYxUmlBa2tndHpEQTFsaGRueDFLSFRCMjZPQmp2V3lxa3NDTVJiCnY4ZElGRUhIRjRCd1ZFUG1jNWxjKzBDVWlmUkVmMjJaRWlpRFlBRjVCYlhpelRsRlRPV0pXaTVDOGJVZFhhZ0sKdi8wYkxxTHlGalBsbGNvMGV4cG1kY0IyS1FaaFdIaU1hTXJMbzZUd28xVkRoSkRjS3Y1V1lOd2JQanhhR1N6Swp5dC8zQlVwa2F6YUs3eXMxVXcwdnhFb1A2aThwQWdNQkFBR2pXVEJYTUE0R0ExVWREd0VCL3dRRUF3SUNwREFQCkJnTlZIUk1CQWY4RUJUQURBUUgvTUIwR0ExVWREZ1FXQkJTcVFZZnV6ZlBTWDkxZllwYW5qeU5RNmFNOGZqQVYKQmdOVkhSRUVEakFNZ2dwcmRXSmxjbTVsZEdWek1BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQ0lGUkpuK3BZRwpxRnY4YXU2eFdwWjl2a2FReC8rS1R3Y0plUytsQWhuVUM5aXpqUGIwMTUxSFQvbXBrcGhubnJvbTZmRDgvSk0rCjU2Q3duT0RJbGRpZytRKzRwZU1uQ1hLdHRMYm5jaGgxcG5RVGJmdmVWMC9FVHVEQmYyVTRhMkFpanJUUEd3NVUKL1prczhJV0ZFcnhvbWtsOWJWKzNBVGo4VVZlT2g5NGNtdFIxMm0xL0VqRldaOXJqMEYzdzREdmgvNmpjcG9EbQp5MEZDQXh5N0RCVGNLTUgveFdMaEFIN3JINDFrNkc4Znp5RStMUDJDbGN5OWowdEJZYm1ZaEZreW9HNEdxbHE2CjVOV1llcEYvU1VGamdtQXNqYlA0dExQT0h2aU1XVU5TdEdkd1NUclFMdHE3TGt6UktjOG9FdEh0Q2p5NDhHUkMKQVVCRzRVNHFhckpWCi0tLS0tRU5EIENFUlRJRklDQVRFLEXAMPLE
```

Comment:
maybe shorten this a bit?

Reply:
same as before, a shorter value is invalid. 
I added EXAMPLE chars at the beginning and the end of the token to clarify it's a fake one.